### PR TITLE
[s] A very very very very VERY important change

### PR DESCRIPTION
**Pull request by Rolo-SegaGenesis**
I think it will change the way TG Station is played, it will change the game to be more focused on roleplaying but it won't be taxating to the players as other heavy rp servers.

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: optional name here
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

### CODE REVIEW

**Comment by kevinz000** [comment]
new phone who dis

File: config/admins.txt [hunk 1]
```diff
@@ -1,134 +1,135 @@
...
+Sweaterkittens = Game Master
+Feemjmeem = Game Master
+JStheguy = Game Master
+excessiveuseofcobby = Game Master
+Plizzard = Game Master
 octareenroon91 = Game Master
-Serpentarium = Game Master
\ No newline at end of file
+Serpentarium = Game Master
+Cazz is a cuck
```
Comment (kevinz000) [inline config/admins.txt]: friend the format is
Cazz = Cuck
"is a" is not a defined operator.

**Comment by UncleBourbon** [comment]
I agree good changes overall, forgot to add the part where cazz can erp anyone anytime and gives him an item that spawns his harem. He is also super sexy best sec ever.

**Comment by kevinz000** [comment]
@LetterN COOL KIDS USE THE DAMN .TXT!!

**Pull request closed without merging**